packages/legacy/core/App/screens/Settings.tsx

```diff
@@ -167,6 +167,11 @@ const Settings: React.FC<SettingsProps> = ({ navigation }) => {
     },
     ...(settings || []),
   ]
+  
+  // Remove the Contact section from Setting per TOKENS.CONFIG
+  if ("true" === `${disableContactsInSettings}`) {
```

Comment:
```suggestion
  if (disableContactsInSettings) {
```

Reply:
Will change it. The bad habit from writing plain JS, worry about entering other strings to make it to be **true**.